dts/include/coronet/a12-cpu.dtsi

```diff
@@ -43,11 +43,51 @@
 				};
 			};
 
-			mv6_port(0, "ethX2" , &aphy0, "gmii");
+			agate_port0: port@0 {
```

Comment:
This hunk is NAK from me and Tobias.  You cannot simply change the `a12-cpu.dtsi` like this, and not in this intrusive manner.   We have used macros before to reduce the size and improve the readability of the dts files, going back to the old ways is not OK.  In particular when what you want is to basically prefix the existing definition, which can be done as easily like this:

    agate_port0: mv6_port(0, "ethX2" , &aphy0, "gmii");

However, what's likely even better is to add a new macro that does something like this:

    mv6_switch_port(agate, 0, "ethX2" , &aphy0, "gmii");

Which could look something like this (untested), in `include/common/mv6.h`:

     #define mv6_label_port(_switch, _reg, _label, _phy, _mode) \
             _switch##_port##_reg: mv6_port(_reg, _label, _phy, mode)

Reply:
No problem, I agree to the critique, In fact, I have prototyped a similar approach as with mv6_label_port, but didn't get it to work (more a problem with the full path reference rather than the macro I believe though). I will give it another try.